File: app/src/main/java/com/neome/api/meta/base/dto/StudioDetails.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Symbol;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDetails extends StudioBase
{
  @Nullable
  public String description;

  @Nullable
  public String label;

  @Nullable
  public StudioModuleSelection modules;

  public Symbol name;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDetails.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioModuleSelection
import com.neome.api.meta.base.Symbol

interface StudioDetails : StudioBase
{
  val description: String?
  val label: String?
  val modules: StudioModuleSelection?
  val name: Symbol
}